src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/EventTypeTest.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.AssignApplicantSolicitorHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.CallbackHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.GeneralApplicationHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.intervener.IntervenerHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import java.io.File;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Slf4j
+public class EventTypeTest {
+
+    private final List<Class> handlerClassesToIgnore = Arrays.asList(FinremCallbackHandler.class, FinremCallbackRequest.class,
+        GeneralApplicationHandler.class, CallbackHandler.class, IntervenerHandler.class, AssignApplicantSolicitorHandler.class);
+
+    @Test
+    public void shouldOnlyBeOneEventHandlerPerEventType() throws ClassNotFoundException {
+        var handlerClasses = getHandlerClasses();
+        List<EventType> eventTypeList = new ArrayList<>(Arrays.stream(EventType.values()).toList());
```

Comment:
I'd suggest doing `for (EventType eventType : EventType.values())` instead and get rid of the variable, same for the other lists below